website/src/contribute/release-guide.md

```diff
@@ -410,7 +383,25 @@ There are 2 ways to perform this verification, either running automation script(
       ```
       ./gradlew build -PisRelease --no-parallel --scan --stacktrace --continue
       ```
-  
+
+
+#### Create release-blocking issues in JIRA
+
+The verify_release_build.sh script may include failing or flaky tests. For each of the failing tests create a JIRA with the following properties:
+
+* Issue Type: Bug
+
+* Summary: Either A) name of failing gradle task IF the majority (>50%) of tests under task are failing or B) name of failing test
```

Comment:
Better to default to listing both, if applicable.

Reply:
Done, also added a proposed format to use when writing the title of the test for uniformity.